.github/workflows/development.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches: [ "master" ]
   workflow_call:
 
+  workflow_dispatch:
```

Comment:
Is this deliberate? I first thought this might not do anything on its own but I see it might let us run the action manually, if that's the idea?
https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow